### DISCUSSION TITLE
Support rem units

### DISCUSSION THREAD

**Discussion by abhay-rana** [nativewind/nativewind#84] [Ideas]
custom font size is giving error it is not able to convert the rem into numbers , but it is working with twrnc 

![image](https://user-images.githubusercontent.com/41327466/174242186-c38cd55e-9dbf-46de-b591-839815f3959c.png)
![image](https://user-images.githubusercontent.com/41327466/174242267-1d788043-5733-4226-8759-251806fb7689.png)
![image](https://user-images.githubusercontent.com/41327466/174242347-be556248-3786-4851-80e0-a8bdd9d38c0b.png)

**Comment by marklawlor** [comment]
Hey thanks for opening the issue. The current version doesn't support non-react-native units, so things like "rem" are not available. You can "polyfill" it by simply creating a helper function within your `tailwind.config.js`

```
const rem = (value, rem = 16) => `${value * rem}px`

fontSize: {
  xs: [rem(0.6875), 'normal]
}
```

I know this takes away the accessibility advantages of `rem`, so in the upcoming `v2` you will be able to use `rem` for web only

```
fontSize: {
  xs: {
    web: ['0.6875rem', 'normal],
    default: [rem(0.6875), 'normal]
}
```

After `v2` I'm going to think more about `rem` and how it can be translated into native. `rem` automatically scaling with the user's font-setting is great for accessibility and might be something we can replicate.

**Comment by vital-tech-results** [comment]
Having rem would be nice. That's a great polyfil suggestion! 

Hello @abhay-rana I wonder how you got your custom colors in tailwind-config.js to work? I [opened an issue](https://github.com/marklawlor/tailwindcss-react-native/issues/80) yesterday as well. If you have a moment, could you provide recommendations there?

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
